scripts/1_clone_cosmos_proto.sh

```diff
@@ -8,23 +8,19 @@ cd "${WD}"
 set -o errexit -o nounset -o pipefail
 command -v shellcheck >/dev/null && shellcheck "$0"
 
-COSMOS_SDK_REF='v0.45.6'
+PROTO_DEPS_DIR="../proto_deps"
+PROTO_DEPS_DIR_FULL_PATH=$(realpath --no-symlink "$PROTO_DEPS_DIR")
+if [ -e "$PROTO_DEPS_DIR_FULL_PATH" ]; then
+  echo "Output path $PROTO_DEPS_DIR_FULL_PATH exists, please remove it first"
+  exit 1
+fi
```

Comment:
Any reason we don't gitignore `proto_deps` if we have this script?

Reply:
I prefer including dependencies so things are more deterministic.
May remove them if you prefer the other way.

Reply:
Does this script always generate same proto or is it dependent on some env?

Reply:
It use `buf` to download the proto files from Cosmos SDK Buf repo: https://buf.build/cosmos/cosmos-sdk
So I guess it falls into `dependent on some env`
That said, given that the commit is fixed (in the `COSMOS_SDK_PROTO_COMMIT` variable in the script), I guess it is always the same